### PR TITLE
[DX] [Admin] Routing and grids sorted alphabetically

### PR DESCRIPTION
**Pull request by lchrusciel**
| Q               | A
| --------------- | ---
| Bug fix?        | no
| New feature?    | no
| BC breaks?      | no
| Deprecations?   | no
| Related tickets | 
| License         | MIT

### CODE REVIEW

File: src/Sylius/Bundle/AdminBundle/Resources/config/routing.yml [hunk 1]
```diff
@@ -10,23 +10,24 @@ sylius_admin_dashboard:
...
+
 sylius_admin_shipping_method:
     resource: "@SyliusAdminBundle/Resources/config/routing/shipping_method.yml"
+
+sylius_admin_tax_category:
+    resource: "@SyliusAdminBundle/Resources/config/routing/tax_category.yml"
+
+sylius_admin_tax_rate:
+    resource: "@SyliusAdminBundle/Resources/config/routing/tax_rate.yml"
+
```
Comment (pamil) [inline src/Sylius/Bundle/AdminBundle/Resources/config/routing.yml]: Two blank lines at the end of file 🐫

**Comment by pjedrzejewski** [comment]
I'd say let's do stuff like that when most grids are done because I am 99% sure that next grids added to this file will not be sorted properly.

**Comment by pjedrzejewski** [comment]
That being said, I will of course merge this PR. :)

**Comment by lchrusciel** [comment]
I have sorted them, to at least have some chance to avoid conflicts. Before, everybody has added theirs resources to the end of list, what force us to make some rebases. I think its worthy to start sorting them.

**Comment by pjedrzejewski** [comment]
👍

**Pull request merged**